<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="gsim_urdf_pkg">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-4.7634E-17 2.2091E-17 0.025"
        rpy="0 0 0" />
      <mass
        value="6.125" />
      <inertia
        ixx="0.063802"
        ixy="-1.1587E-17"
        ixz="-4.2248E-19"
        iyy="0.063802"
        iyz="7.4848E-35"
        izz="0.12505" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="first_link">
    <inertial>
      <origin
        xyz="0.060001 0.073 0.17496"
        rpy="0 0 0" />
      <mass
        value="5.2657" />
      <inertia
        ixx="0.012714"
        ixy="-1.2779E-07"
        ixz="-4.0422E-08"
        iyy="0.0098435"
        iyz="-1.3314E-08"
        izz="0.015733" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/first_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/first_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_First"
    type="fixed">
    <origin
      xyz="-0.051662 -0.082969 0.05"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="first_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="second_link">
    <inertial>
      <origin
        xyz="0.060001 0.075 0.17496"
        rpy="0 0 0" />
      <mass
        value="5.2657" />
      <inertia
        ixx="0.012714"
        ixy="-1.2779E-07"
        ixz="-4.0422E-08"
        iyy="0.0098435"
        iyz="-1.3314E-08"
        izz="0.015733" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/second_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/second_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_Second"
    type="prismatic">
    <origin
      xyz="0 -0.002 0.35"
      rpy="0 0 0" />
    <parent
      link="first_link" />
    <child
      link="second_link" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-0.1"
      upper="0.1"
      effort="10"
      velocity="2" />
  </joint>
  <link
    name="third_link">
    <inertial>
      <origin
        xyz="0.060001 0.075 0.17496"
        rpy="0 0 0" />
      <mass
        value="5.2657" />
      <inertia
        ixx="0.012714"
        ixy="-1.2779E-07"
        ixz="-4.0422E-08"
        iyy="0.0098435"
        iyz="-1.3314E-08"
        izz="0.015733" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/third_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/third_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_Third"
    type="prismatic">
    <origin
      xyz="0 0 0.35"
      rpy="0 0 0" />
    <parent
      link="second_link" />
    <child
      link="third_link" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-0.1"
      upper="0.1"
      effort="10"
      velocity="2" />
  </joint>
  <link
    name="fourth_link">
    <inertial>
      <origin
        xyz="0.060001 0.074 0.17496"
        rpy="0 0 0" />
      <mass
        value="5.2657" />
      <inertia
        ixx="0.012714"
        ixy="-1.2779E-07"
        ixz="-4.0422E-08"
        iyy="0.0098435"
        iyz="-1.3314E-08"
        izz="0.015733" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/fourth_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/fourth_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_Fourth"
    type="prismatic">
    <origin
      xyz="0 0.001 0.35"
      rpy="0 0 0" />
    <parent
      link="third_link" />
    <child
      link="fourth_link" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-0.1"
      upper="0.1"
      effort="10"
      velocity="2" />
  </joint>
  <link
    name="fifth_link">
    <inertial>
      <origin
        xyz="0.060001 0.075 0.17496"
        rpy="0 0 0" />
      <mass
        value="5.2657" />
      <inertia
        ixx="0.012714"
        ixy="-1.2779E-07"
        ixz="-4.0422E-08"
        iyy="0.0098435"
        iyz="-1.3314E-08"
        izz="0.015733" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/fifth_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.75294 0.75294 0.75294 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://gsim_urdf_pkg/meshes/fifth_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_Fifth"
    type="prismatic">
    <origin
      xyz="0 -0.001 0.35"
      rpy="0 0 0" />
    <parent
      link="fourth_link" />
    <child
      link="fifth_link" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-0.1"
      upper="0.1"
      effort="10"
      velocity="2" />
  </joint>
</robot>